src/main/scala/mesosphere/marathon/api/akkahttp/v2/TasksController.scala

```diff
@@ -48,7 +53,10 @@ class TasksController(
           pathEndOrSingleSlash {
             listTasks()
           }
-        }
+        } ~
+          (path("delete") & post) {
```

Comment:
Out of curiosity, why did you opt for `(path("delete") & post)` instead of `post { path("delete") }` as done for `get`?

Reply:
Good question - It was the pattern that I have seen in few other controllers so I just stick with it. I also don't like deep nesting...